src/server/network/protocol/protocolgame.cpp

```diff
@@ -73,13 +73,22 @@ void ProtocolGame::AddItem(NetworkMessage &msg, uint16_t id, uint8_t count)
 	if (it.isPodium) {
 		msg.add<uint16_t>(0);
 		msg.add<uint16_t>(0);
+		msg.add<uint16_t>(0);
 
 		msg.addByte(2);
 		msg.addByte(0x01);
 	}
 	if (it.upgradeClassification > 0) {
 		msg.addByte(0);
 	}
+	if (it.expire || it.expireStop || it.clockExpire) {
+		msg.add<uint32_t>(it.decayTime);
+		msg.addByte(0x01); // Unknown byte
```

Comment:
figured it out:
settings -> interface -> show expiry time on unused items

0x00 - already used
0x01 - default charges/brand-new